### DISCUSSION TITLE
Stripe Payment Provider not seeing in admin side

### DISCUSSION THREAD

**Discussion by MohammedHafiz** [medusajs/medusa#2915] [Q&A]
I am using stripe payment integration in my medusa project and  I integrated stripe according to the documentation. But for the admin part, in the region we are selecting the payment provider, here I cannot see stripe as an option, all I can see is one option 'manual'. Could you please explain to me how to get the stripe option on admin side

**Comment by olivermrbl** [comment]
Can I get you to ensure, you've performed the following steps:
- Installed the Stripe plugin (`medusa-payment-stripe`) via npm
```
yarn add medusa-payment-stripe@latest
```
- Added the Stripe plugin configuration in your `medusa-config.js`
```ts
{
    resolve: `medusa-payment-stripe`,
    options: {
      api_key: "api_key",
      webhook_secret: "wh_secret,
    },
  },
```

Stripe will only appear in Medusa Admin, if these steps are configured correctly.

**Comment by MohammedHafiz** [comment]
Yes I have added that

**Comment by pKorsholm** [comment]
Hi @MohammedHafiz 

A couple of follow up questions: 
- what versions of medusa are you running? 
- Can you paste your server log when running medusa?
- Just confirming you have added medusa-stripe to your plugin array as so? 
```
const plugins = [
  `medusa-fulfillment-manual`,
  ...
  {
    resolve: `medusa-payment-stripe`,
    options: {
      api_key: STRIPE_API_KEY,
      webhook_secret: STRIPE_WEBHOOK_SECRET,
    },
  },
```

**Reply by Gunkev** [reply]
@MohammedHafiz  and @pKorsholm  I am facing the same issue I am getting a 404 when testing `stripe trigger payment_intent.succeeded`
I have `Medusa CLI version: 1.3.6`

Here is my log
![not found](https://user-images.githubusercontent.com/36812576/227367291-97bf0771-45fb-49bf-870d-6c27bcd7fd1f.PNG)


And yes. I have added the stripe plugin and configuration per the documentation

**Comment by flashjames** [comment]
I have the same problem, I followed these guides:

- https://docs.medusajs.com/plugins/payment/stripe
- https://docs.medusajs.com/modules/carts-and-checkout/backend/add-payment-provider

And I get stuck here:
https://docs.medusajs.com/user-guide/regions/providers#manage-payment-providers

As stripe payment provider isnt listed there, only manual.

And this is how admin looks:
![image](https://user-images.githubusercontent.com/359702/227456083-eaf3163b-83d9-4c91-b5c0-812b4134f588.png)

## Medusa versions
- Medusa backend: Tried with the @latest and the current master of medusa
- Medusa admin: Tried with @lastest and the current master branch

Note: Will be investigating were the issue is right now.


> Can I get you to ensure, you've performed the following steps:
> 
>     * Installed the Stripe plugin (`medusa-payment-stripe`) via npm
> 
> 
> ```
> yarn add medusa-payment-stripe@latest
> ```
> 
>     * Added the Stripe plugin configuration in your `medusa-config.js`
> 
> 
> ```ts
> {
>     resolve: `medusa-payment-stripe`,
>     options: {
>       api_key: "api_key",
>       webhook_secret: "wh_secret,
>     },
>   },
> ```
> 
> Stripe will only appear in Medusa Admin, if these steps are configured correctly.
Yes I've followed those.


Edit: Have made an issue for this: https://github.com/medusajs/medusa/issues/3565

**Comment by flashjames** [comment]
I solved it, see answer in the issue I made: https://github.com/medusajs/medusa/issues/3565#issuecomment-1482526982

Bascially get latest admin repo code, and note that nowadays "main" seem to be the default branch, and not master.

@Gunkev @MohammedHafiz

**Reply by Gunkev** [reply]
@flashjames I have the latest version of medusa admin. 
`Author: Oliver Windall Juhl <59018053+olivermrbl@users.noreply.github.com>`
`Date:   Mon Mar 6 12:55:47 2023 +0100`

`fix: Discount in DraftOrder create flow (#905)`

But I am still not able to see the payment provider on my dashbaord

**Reply by Gunkev** [reply]
@flashjames please could you reproduce the steps you used? I just checked my admin repo and it's up to date with the remote one

**Reply by hosenur** [reply]
Hey @Gunkev , I am facing this same issue , I am also on the latest version , did you find a fix for this ?

**Comment by luie760** [comment]
I know this is from a year ago, but I thought I should post what worked for me here since I tried everything that was mentioned in this post. I do have to mention that I am using Medusa v2. 

My issue was in the `medusa-config.ts` file. Medusa v2 payment providers are no longer configured under the plugins array. You need to configure them under the modules array now. 

```ts
import { defineConfig, loadEnv } from "@medusajs/framework/utils";

loadEnv(process.env.NODE_ENV || "development", process.cwd());

export default defineConfig({
  projectConfig: {
    databaseUrl: process.env.DATABASE_URL,
    redisUrl: process.env.REDIS_URL || "",
    http: {
      storeCors: process.env.STORE_CORS!,
      adminCors: process.env.ADMIN_CORS!,
      authCors: process.env.AUTH_CORS!,
      jwtSecret: process.env.JWT_SECRET || "supersecret",
      cookieSecret: process.env.COOKIE_SECRET || "supersecret",
    },
  },
  featureFlags: {
    medusa_ff_disable_admin: true,
  },
  modules: [
    {
      resolve: "@medusajs/medusa/payment",
      options: {
        providers: [
          {
            resolve: "@medusajs/medusa/payment-stripe",
            id: "stripe",
            options: {
              apiKey: process.env.STRIPE_SECRET_KEY,
              webhookSecret: process.env.STRIPE_WEBHOOK_SECRET,
              capture: true,
            },
          },
        ],
      },
    },
  ],
});
```

After setting my config file like this and restarting my dev server, I was able to see the Stripe providers in my admin dropdown for payment providers!!
